barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.hpp

```diff
@@ -56,6 +56,8 @@ class UltraCircuitBuilder_ : public CircuitBuilderBase<typename Arithmetization:
     // number of gates created per non-native field operation in process_non_native_field_multiplications
     static constexpr size_t GATES_PER_NON_NATIVE_FIELD_MULTIPLICATION_ARITHMETIC = 7;
 
+    size_t num_vars_added_in_constructor; // needed in constructing circuit from acir
```

Comment:
nit: could have an = 0 for sanity just in case code gets deleted

Reply:
done